tests/functional/u/unidiomatic_typecheck.py

```diff
@@ -61,10 +61,22 @@ def deliberate_subclass_check_negatives(b):
     type(42) is type(b)
     type(42) is not type(b)
 
-def type_of_literals_positives(a):
-    type(a) is type([]) # [unidiomatic-typecheck]
-    type(a) is not type([]) # [unidiomatic-typecheck]
-    type(a) is type({}) # [unidiomatic-typecheck]
-    type(a) is not type({}) # [unidiomatic-typecheck]
-    type(a) is type("") # [unidiomatic-typecheck]
-    type(a) is not type("") # [unidiomatic-typecheck]
+def type_of_literals_negatives(a):
+    type(a) is type([])
+    type(a) is not type([])
+    type(a) is type({})
+    type(a) is not type({})
+    type(a) is type("")
+    type(a) is not type("")
+    type(a) == type([])
```

Comment:
Doesn't the issue say that we don't want to suggest this but only exclude the `is` and `is not` comparisons?

Reply:
Let's discuss in https://github.com/pylint-dev/pylint/pull/10170#discussion_r1938498563 if required.